docs/resources/azure_sentinel_alert_rule.md

```diff
@@ -0,0 +1,120 @@
+---
+title: About the azure_sentinel_alert_rule Resource
+platform: azure
+---
+
+# azure_sentinel_alert_rule
+
+Use the `azure_sentinel_alert_rule` InSpec audit resource to test properties related to alert_rule for a resource group or the entire subscription.
+
+## Azure REST API version, endpoint, and HTTP client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the HTTP client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+For api related info : [`Azure alert_rule Docs`](https://docs.microsoft.com/en-us/rest/api/securityinsights/stable/alert-rules/get).
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_sentinel_alert_rules` resource block returns all Azure alert_rule, either within a Resource Group (if provided), or within an entire Subscription.
```

Comment:
Is `resource_group` required? It doesn't look like it, but it says it's required on line 39.